implementations/micrometer-registry-dynatrace/src/test/java/com/dynatrace/file/util/FileBasedConfigurationTestHelper.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2021 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.dynatrace.file.util;
+
+public class FileBasedConfigurationTestHelper {
+    // This allows overwriting the file location of the file-based configuration for testing.
+    public static void forceOverwriteConfig(String filename) {
+        DynatraceFileBasedConfigurationProvider.getInstance().forceOverwriteConfig(filename);
```

Comment:
I've just realized why is this needed.
Is it possible to make it public or release a test dependency where this is available?